components/device-mgt/org.wso2.carbon.device.mgt.common/src/main/java/org/wso2/carbon/device/mgt/common/group/mgt/GroupAlreadyExistException.java

```diff
@@ -21,31 +21,31 @@
 /**
  * This class represents a custom exception specified for group management
  */
-public class GroupAlreadyEixistException extends Exception {
+public class GroupAlreadyExistException extends Exception {
 
     private static final long serialVersionUID = -312678379574816874L;
     private String errorMessage;
 
-    public GroupAlreadyEixistException(String msg, Exception nestedEx) {
+    public GroupAlreadyExistException(String msg, Exception nestedEx) {
         super(msg, nestedEx);
         setErrorMessage(msg);
     }
 
-    public GroupAlreadyEixistException(String message, Throwable cause) {
+    public GroupAlreadyExistException(String message, Throwable cause) {
         super(message, cause);
         setErrorMessage(message);
     }
 
-    public GroupAlreadyEixistException(String msg) {
+    public GroupAlreadyExistException(String msg) {
```

Comment:
Ack, Fixed in commit https://github.com/wso2/carbon-device-mgt/pull/305/commits/c19ddcfe1549144e93e3781ec73bc8be1c6b2da3